test/mksymbols.rb

```diff
@@ -42,4 +42,3 @@
 #{o}
 ];
 FOO
-
```

Comment:
Use the git command line and remove it from the patch.  http://stackoverflow.com/questions/5020305/how-can-i-remove-a-change-in-1-file-in-1-commit